### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by Amar2076**
git tutorials changes

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -106,7 +106,7 @@ class _MyHomePageState extends State<MyHomePage> {
           mainAxisAlignment: MainAxisAlignment.center,
           children: <Widget>[
             const Text(
-              'You have pushed the button this many times:',
+              'YhcguwUwiuwi:',
```
Comment (viji1008) [inline lib/main.dart:109]: Please remove the unwanted text

Comment (sivakarthikayancs) [inline lib/main.dart:109]: Give proper Text

**Comment by Dhana-palan** [comment]
changes r commented with changes

**Review by rajeshwarig12345 — COMMENTED** [review]
remove the unwanted files

**Review by viji1008 — APPROVED** [review]

**Review by Dhana-palan — APPROVED** [review]

**Pull request merged**